dotnet/src/Connectors/Connectors.HuggingFace/Connectors.HuggingFace.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.HuggingFace</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Connectors.HuggingFace</RootNamespace>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
```

Comment:
Is the expectation that this will be its own nuget package? Should we add the nuget.props now?